src/commands/synthetics/__tests__/run-tests-lib.test.ts

```diff
@@ -476,20 +476,39 @@ describe('run-test', () => {
       })
```

Comment:
Do we also want a test on return values, or is TS signature enough?

Could add something like 
```
    test('should return result values returned by executeTests', async () => {
      const returnValue = await runTests.executeWithDetails({}, {})
      expect(returnValue.results).toBeDefined()
      expect(returnValue.summary).toBeDefined()
    })
]
```